src/main/java/examples/Examples.java

```diff
@@ -99,7 +100,8 @@ public void secure(Vertx vertx) {
       .setAddress("database-service-address")
       // Secure the messages in transit
       .addInterceptor(
```

Comment:
the API should not make appear the holder, instead we should overload `addInterceptor` with an extra name parameter so it would look like:

```java
addInterceptor("action", ...).register(...);
```

Reply:
updated